package.sh

```diff
@@ -2,6 +2,11 @@
 
 chart="${1:?CHART NAME REQUIRED}"
 
-helm dependency update $chart
-helm package $chart --destination docs
-helm repo index docs
+mkdir tmp_charts
```

Comment:
Should we use something like mktmp here so if it fails it doesnt break the next time it runs locally?  mkdir -p could also work but wouldnt make sure the directory was always empty.